kolibri/core/templates/kolibri/unsupported_browser.html

```diff
@@ -0,0 +1,34 @@
+{% load i18n %}
+<style>
+  body {
```

Comment:
Just a bit of css cleanup: 
```css
<style>
  body {
    font-family: sans-serif;
    margin: 0;
    background-color: white;
  }
  .topbar {
    background-color: #996189;
    width: 100%;
    color: white;
  }
  .topbar h1 {
    font-size: x-large;
    display: inline-block;
    padding-left: 32px;
  }
  .content-container {
    padding: 32px;
  }
</style>
```
![localhost_8000_user_](https://user-images.githubusercontent.com/7193975/33152823-92bd87ea-cf93-11e7-896d-51641c33faa7.png)

Reply:
For reference, the before:
![localhost_8000_user_ 1](https://user-images.githubusercontent.com/7193975/33152854-b8925c20-cf93-11e7-90f9-37b13bd0da70.png)

@jtamiace Changes look good?

Reply:
Yes 👍

Reply:
Suggestion:
```
The browser or browser version you are using is not supported. 
Update your browser, or try Firefox or Chrome which are recommended for Kolibri.
```

Reply:
Oh, shouldn't have just copied styling verbatim. @christianmemije the reason for the symmetrical padding was to ensure that the page is RTL compatible without any dynamic styling needed.